distribution/debian/postinst

```diff
@@ -10,24 +12,14 @@ if ! getent passwd "$USER" >/dev/null; then
     usermod -a -G "$USER" "$USER"
   fi
 fi
-  
-# Create home directory
 
+# Create home directory
 if [ ! -d /var/opt/sonarr ]; then
   mkdir -p /var/opt/sonarr
-  chown $USER:$USER /var/opt/sonarr
+  chown -R $USER: /var/opt/sonarr
 fi
 
 # Set permissions on /opt/sonarr
+chown -R $USER: /opt/sonarr
 
-chown -R $USER:$USER /opt/sonarr
-
-# Start Sonarr automatically
-
-if [ -f /proc/1/comm ]; then
-  if [ "`cat /proc/1/comm`" = "systemd" ]; then
-    # Enable and start systemd service
-    systemctl enable sonarr.service
-    systemctl start sonarr.service
-  fi
-fi
+#DEBHELPER#
```

Comment:
This special syntax?

Reply:
Get's replaced by the code automatically generated by dh_* (dh_systemd mostly in this case).

Example:
```
chown -R $USER: /opt/sonarr

# Automatically added by dh_systemd_enable
# This will only remove masks created by d-s-h on package removal.
deb-systemd-helper unmask sonarr.service >/dev/null || true

# was-enabled defaults to true, so new installations run enable.
if deb-systemd-helper --quiet was-enabled sonarr.service; then
	# Enables the unit on first installation, creates new
	# symlinks on upgrades if the unit file has changed.
	deb-systemd-helper enable sonarr.service >/dev/null || true
else
	# Update the statefile to add new symlinks (if any), which need to be
	# cleaned up on purge. Also remove old symlinks.
	deb-systemd-helper update-state sonarr.service >/dev/null || true
fi
# End automatically added section
# Automatically added by dh_systemd_start
if [ -d /run/systemd/system ]; then
	systemctl --system daemon-reload >/dev/null || true
	deb-systemd-invoke start sonarr.service >/dev/null || true
fi
# End automatically added section
```